---
layout: page
title: Hiring!
description: Multiple student projects available
img: assets/img/uncleSam.jpg
importance: 1
category: Student projects
#related_publications: einstein1956investigations, einstein1950meaning
---

There is a number of theoretical projects open for students interested in any of the research topics (both at BSc and MSc level). Some of the specific openings will be posted below. 

Internships involve multiple modeling types (genome-scale metabolic modelling and kinetic modelling) and rely on programming with Python, MATLAB and/or R for data analysis and visualization. Obviously, you are also welcome to bring any interesting reseach question yourself - we love puzzles at the Systems Biology Lab! Keep in mind that the students from the Vrije Universiteit have priority in hiring, although students from other universities/HBO are also open for consideration. 
Please contact me for more details!

#### Open positions
##### Exploring the diversity and origins of metabolites in ant social fluids
Research internship proposal (MSc Minor/Major, 24-36 EC)
Supervisor: Pranas Grigaitis

Many insect orders, and most ant species, exhibit eusocial behavior, i.e., individuals live in groups, coordinated through cooperative action and reproductive division of labor. One of manifestations of eusociality is presence of socially exchanged fluids. Our collaborators (lab of [Adria LeBoeuf @Uni Cambridge](https://leboeuflab.com/)) have previously observed that the trophallactic (=nutritional) fluids of Florida carpenter ants *Camponotus floridanus* are likely rich in sugars and free fatty- and amino acids ([LeBoeuf *et al.* 2016 *eLife*](https://elifesciences.org/articles/20375, [Hakala et *al. 2021* *eLife*](https://elifesciences.org/articles/74005)).

The LeBoeuf lab has performed a large-scale sampling experiment of social fluids from 4 different ant species grown in lab, and multiple castes/fluid localizations within a species. We have run LC-MS/MS-based untargeted metabolomics analyses on these samples and acquired mass spectra, containing >1500 unique peaks. The identity of >90% peaks, however, is ambiguous and needs further investigation. We are looking for a student to analyze these metabolomics datasets, and, coupled with genomic analyses, help us to suggest the origin of secondary metabolites (ant- or microbe-produced, or food-borne) in different ant social fluids.

We will use mzMine3 ([Heuckeroth *et al.* 2024 *Nat Protoc*](https://doi.org/10.1038/s41596-024-00996-y)) and other well-established mass spectrometry analysis tools to reannotate peaks from the raw mass spectra, and assess the diversity of fluid metabolites at different granularity (from functional classes towards individual compounds). For mapping the origin of these metabolites, we will use various genome annotation tools to locate enzymes and/or biosynthetic gene clusters. 

We expect you to do scripting in either *Python/R/MATLAB*; basic knowledge of *bash* scripting is an advantage. A background in molecular biosciences is desired, but strongly interested applicants from other backgrounds should apply as well. We offer you a challenge to analyze -omics datasets from unique biological sources, and to join an exciting interdisciplinary project where you will join a diverse team of ecologists, molecular zoologists, and systems biologists.


##### Correlating genetic and functional variability of enzymes with thermodynamics
MSc Minor/Major, 24-36 EC | both on-site and remote | availability: Summer 2024

Supervisors: Pranas Grigaitis, Frank Bruggeman

Most biochemical processes require protein catalysts – enzymes – to proceed at rates which can sustain life. The catalysis function of these proteins has evolved under (and is still subject to) natural selection as different variants of the enzymes emerge through genetic mutations. Mutations can bring both innovations (improved or novel function) and detrimental damage (loss of function) to the enzymes and thus directly influence cell fitness. We would like to test the hypothesis whether the genes, encoding enzymes which catalyze reactions close to thermodynamic equilibrium (small \|ΔG\| values) are less prone to accumulate mutations in selected microbial species.

In this project, we want to correlate the functional information of enzymes with their mutational patterns at different levels (gene, transcript, protein) and expression of these enzymes. We have previously developed a routine to compute ΔG values for all known enzyme-catalyzed metabolic reactions from the genome-scale metabolic models. We will collect and use the set of gene-, transcript-, and protein sequences of sequenced microbial isolates and absolute proteome quantification data for reference strains to correlate sequence variability and enzyme expression levels with the thermodynamic parameters. 

We expect you to be experienced in Python scripting and dedicated data handling libraries (pandas, numpy, scipy). Good understanding of core statistical analyses will be needed for analyzing data. Previous experience with setting up command-line bioinformatics routines is an advantage. We offer you an opportunity to gain experience in handling large-scale datasets, and learn how to setup and run complex, multi-step bioinformatics pipelines.

